staging/src/k8s.io/api/authentication/v1alpha1/types.go

```diff
@@ -28,7 +28,8 @@ import (
 // +k8s:prerelease-lifecycle-gen:introduced=1.25
 
 // SelfSubjectReview contains the user information that the kube-apiserver has about the user making this request.
-// When using impersonation, users will receive the user info of the user being impersonated.
+// When using impersonation, users will receive the user info of the user being impersonated.  If impersonation or
```

Comment:
the impersonation clarification is useful... the `extra keys are case-ignored/lowercased` seems like a specific aspect of `the user information that the kube-apiserver has about the user making this request`... I'm not sure why that is any more important to include here than things like auto-added groups when impersonating a service account

Reply:
Some providers like EKS use extra keys with capital letters in them.  I was trying to describe "if you try to impersonate an identity like that, it won't roundtrip correctly and you will see lowercase - it's not a bug it's just how the headers work."  This might be too niche to matter.

Reply:
doesn't that mean those also won't translate to aggregated servers? extra keys should use lowercase characters

Reply:
> doesn't that mean those also won't translate to aggregated servers? extra keys should use lowercase characters

Indeed they don't.  But it's likely too late for EKS to fix that...